blogposts/2022/release-3.36.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Sourcegraph 3.65 release'
+publishDate: 2022-01-21T10:00-07:00
+description: 'Sourcegraph 3.36 introduces...'
+tags: [blog, release]
+slug: 'release/3.36'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.36/sourcegraph-3-36-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.36/sourcegraph-3-36-release.png
+changelogItems:
+---
+
+Sourcegraph 3.36 is now available! Here are some highlights from this release:
+
+## Batch Changes on forks
+
+Before this release, creating a batch change required you (or your service-account) to have write-access to every target repository. This was incompatible with having tight repository access control, and was impractical at a large scale; you want all users to be able to create batch changes on all repositories, but also not grant them overly broad write permissions.
+
+That's why we are introducing [batch changes on forks](https://docs.sourcegraph.com/admin/config/batch_changes), which makes it possible for branches created by Batch Changes to be pushed to a fork of the repository instead of a branch. This solves those access control challenges and lets you safely enable Batch Changes for everyone. In this version, this is an instance setting that is either on for all users, or off (default).
```

Comment:
@malomarrec I think this may be a little confusing as written, where you say "makes it possible for branches created by BC to be pushed to a fork of the repository instead of a branch". Does this suggestion make sense?

```suggestion
That's why we are introducing [batch changes on forks](https://docs.sourcegraph.com/admin/config/batch_changes), which makes it possible for branches created by Batch Changes to be pushed to a fork of the repository instead of the repository itself. This solves those access control challenges and lets you safely enable Batch Changes for everyone. In this version, this is an instance setting that is either on for all users, or off (default).
```